src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -149,19 +147,9 @@ public boolean isSameInterview(Interview otherInterview) {
      */
     public boolean isValid() {
         return startTime.isBefore(endTime)
-                && isWithinWorkingHours(startTime)
-                && isWithinWorkingHours(endTime)
-                && startTime.toLocalDate().isEqual(endTime.toLocalDate());
-    }
-
-    /**
-     * Returns true if startTime and endTime are within working hours,
-     * which is defined to be between 0900 and 1700.
-     */
-    public boolean isWithinWorkingHours(LocalDateTime dateTime) {
-        LocalTime time = dateTime.toLocalTime();
-        return (time.isAfter(WORK_START) || time.equals(WORK_START))
-                && (time.isBefore(WORK_END) || time.equals(WORK_END));
+                && startTime.isWithinWorkingHours()
+                && endTime.isWithinWorkingHours()
+                && startTime.getTime().equals(endTime.getTime());
```

Comment:
I think this line should check if the dates are the same and not the time. The start time and end time are valid only if the date for both are the same.

Reply:
Good catch! My apologies for the carelessness! 👍 
Adding commit to this PR branch to fix it right now.